src/directsketch.rs

```diff
@@ -167,7 +176,7 @@ async fn download_with_retry(
                         return Ok(data.to_vec());
                     } else {
                         last_error = Some(anyhow!(
-                            "MD5 hash does not match. Expected: {}, Found: {}",
+                            "MD5 hash does not match. Expected: {}; Found: {}",
```

Comment:
might be good to put quotes in here?

```suggestion
                            "MD5 hash does not match. Expected: '{}'; Found: '{}'",
```

Reply:
oh, never mind, not for an md5. Duh.

Reply:
good idea, added in https://github.com/sourmash-bio/sourmash_plugin_directsketch/pull/92/commits/139c956a41b5e91d9d41c61ab00d3a55f0c6009a!

Reply:
whoops, i missed your second comment somehow. I can roll it back if you think it's better without 🤷🏻‍♀️

Reply:
oh it's fine there, I just wanted it ALSO in the other error message, `"Invalid checksum line format in URL {}: {}` :)

Reply:
hah, I should really refresh this page more often when we're doing this in real time. On it 😂